src/main/scala/fi/oph/koski/oppija/KoskiOppijaFacade.scala

```diff
@@ -19,8 +19,8 @@ import fi.oph.koski.util.{Timing, WithWarnings}
 class KoskiOppijaFacade(henkilöRepository: HenkilöRepository, henkilöCache: KoskiHenkilöCache, opiskeluoikeusRepository: CompositeOpiskeluoikeusRepository, historyRepository: OpiskeluoikeusHistoryRepository, perustiedotIndexer: OpiskeluoikeudenPerustiedotIndexer, config: Config, hetu: Hetu) extends Logging with Timing with GlobalExecutionContext {
   private lazy val mockOids = config.hasPath("authentication-service.mockOid") && config.getBoolean("authentication-service.mockOid")
 
-  def findOppija(oid: String, useVirta: Boolean = true, useYtr: Boolean = true)(implicit user: KoskiSession): Either[HttpStatus, WithWarnings[Oppija]] = {
-    henkilöRepository.findByOid(oid)
+  def findOppija(oid: String, withLinkedOids: Boolean = false, useVirta: Boolean = true, useYtr: Boolean = true)(implicit user: KoskiSession): Either[HttpStatus, WithWarnings[Oppija]] = {
```

Comment:
Hmm, mikähän olisi tässä oikea nimi tuolle withLinkedOids parametrille.... nimittäin jos annettu oid on master, niin tämä kutsu näyttäisi palauttavan myös slave-oidien tiedot, vaikka annettaisiin withLinkedOids = false.